data/LSIR/projects.yaml

```diff
@@ -67,75 +77,85 @@ projects:
     information:
       - type: Demo
         title: Online Technology Demos
-        url: https://modemos.epfl.ch/
-      - type: Demo
-        title: Online Image Retrieval Demo
-        url: https://modemos.epfl.ch/article
+        url: https://modemos.epfl.ch/main
     contacts:
-      - name: Rémi Philippe Lebret
+      - name: Rémi Lebret
         email: remi.lebret@epfl.ch
+      - name: Hamza Harkous
+        email: hamza.harkous@gmail.com
     tags:
       - Image Classification
       - Test Classification
       - Artificial Intelligence
     type: Framework
     date_added: 2019-03-18
-    date_updated: 2021-01-22
+    date_updated: 2021-04-22
 
   ai-data-platform-v2:
     name: AI-Driven Classifier Building Pipeline
     description: Platform for speeding up the development of Artificial-Intelligence classifiers
-    url: https://www.epfl.ch/labs/lsir/ai-driven-classifier-building-pipeline/
```

Comment:
That's a matter for a long debate, whether to end URLs with a slash or not  ;-)

Reply:
Héhé, indeed it is, as you saw, I prefer not having slash at the end, but it's clearly a matter of taste :)